hono/test/items.test.ts

```diff
@@ -1,16 +1,48 @@
-// import app from '../src/index';
-//
-// const MOCK_ENV = {
-//  DB: {
-//    prepare: () => {
-//      // Mocked D1
-//    },
-//  },
-// };
+import app from '../src/index'
+
+interface Item {
```

Comment:
Hmmmm declaring this here seems like a problem if the actual db schema is updated (we'd have to update constraints in 2 spots). Ideally we'd also need to write unit tests for the database schema itself (try to create a new item in the database and omit the authorUsername field, it should throw because of the `notNull()` constraint). With this implementation, testing the schema itself seems impossible.

Is there a way to get miniflare to have a mini D1 running???